fleetctl/journal.go

```diff
@@ -46,6 +47,8 @@ func init() {
 	cmdJournal.Flags.IntVar(&flagLines, "lines", 10, "Number of recent log lines to return")
 	cmdJournal.Flags.BoolVar(&flagFollow, "follow", false, "Continuously print new entries as they are appended to the journal.")
 	cmdJournal.Flags.BoolVar(&flagFollow, "f", false, "Shorthand for --follow")
+	cmdJournal.Flags.BoolVar(&flagSudo, "sudo", false, "Execute journal command with sudo")
+	cmdJournal.Flags.BoolVar(&flagSudo, "s", false, "Shorthand for --sudo")
```

Comment:
Let's not add the shortened version of the flag until we have a library to help make it less cumbersome.